src/app/help-panel/help-panel.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-help-panel',
+  templateUrl: './help-panel.component.html',
+  styleUrls: ['./help-panel.component.scss']
+})
+export class HelpPanelComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
I removed `ngOnInit () {} ` because it was void, but now I get this error: `Property 'ngOnInit' is missing in type 'HelpPanelComponent' but required in type 'OnInit'`